package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "pact-gen-ts",
-    "version": "0.9.0",
+    "version": "0.9.1",
```

Comment:
You have to update version to 0.9.2.

There has been already 0.9.1 version

Reply:
Semantic-versioning library is super convenient :) - I guess we could think about implementing it.